.github/workflows/post_publish.yml

```diff
@@ -51,3 +51,52 @@ jobs:
     if: ${{ github.event.workflow_run.conclusion == 'failure' }}
     steps:
       - run: echo 'The triggering workflow failed'
+
+
+  registry-check:
+    # strategy:
+    #   matrix:
+    #     os: [ubuntu-latest, windows-latest, macOS-latest]
+    # runs-on: ${{ matrix.os }}
+    # runs-on: ubuntu-latest
+    runs:
+      using: "composite"
```

Comment:
These are only needed for composite actions, so you'll need to remove them since this is a "normal" workflow

```suggestion
```